pkg/controller/plan/adapter/base/doc.go

```diff
@@ -27,6 +27,10 @@ const (
 	// Set the source PVC of the conversion, used on the DV for filtering
 	AnnConversionSourcePVC = "forklift.konveyor.io/conversionSourcePVC"
 
+	// Annotation that provides a template for customizing persistent volume claim names
+	// during the migration process. The template can reference VM and Plan fields.
+	AnnForkliftPVCNameTemplate = "forklift.konveyor.io/pvc-name-template"
```

Comment:
Why do we need the annotation? Can't we get the information directly form CR?

Reply:
yes, we can ... woo, will change and ping here ...

Reply:
updated the code